includes/import-functions.php

```diff
@@ -45,7 +45,7 @@ function edd_show_coupon_upload_form() {
 			</tr>
 			<tr valign="top">
 				<th scope="row"><?php _e('Validate File Only','edd') ?></th>
-				<td><input type="checkbox" id="check_file" name="check_file" /> <?php _e('Check file, but do not save coupons to the database.','edd') ?></td>
+				<td><input type="checkbox" id="check_file" name="check_file" value="on" /> <?php _e('Check file, but do not save coupons to the database.','edd') ?></td>
```

Comment:
```suggestion
				<td><input type="checkbox" id="check_file" name="check_file" value="on" /> <label for="check_file"><?php esc_html_e( 'Check file, but do not save coupons to the database.', 'edd' ); ?></label></td>
```
It would be nice to go ahead and get the label set up for the checkbox--I didn't fix the text domain as I know you've already opened an issue for it.

Reply:
Fixed in 9f6d8857702fb529caa4447d013f53616fe23cda.